debian/changelog

```diff
@@ -1,3 +1,92 @@
+snapcraft (2.0) xenial; urgency=medium
+
+  [ Daniel Holbach ]
+  * add document detailing the snapcraft.yaml syntax.
+  * fix typo - thanks Kyle.
+  * add missing periods.
+  * add missing 'to'.
+  * rephrase sentence based on Leo's feedback.
+  * remove vendor for 'master'.
+  * add document about parts.
+  * add link to config docs.
+
+  [ Sergio Schvezov ]
+  * Removing the `run` command.
+  * Update the release for the tomcat example snap.
+  * Wrapping exe should not be over match to replace.
+  * cli: Move to docopt, help cmd migrated.
+  * cli: list-plugins cmd migrated to docopts.
+  * cli: clean cmd migrated to docopts.
+  * cli: init cmd migrated to docopts.
+  * cli: pull command migrated to docopts.
+  * clean cmd: migrate to use new lifecycle attribs.
+  * Removing dead test code.
+  * cli: build command migrated to docopts.
+  * cli: stage command migrated to docopts.
+  * Remove dead code from lifecycle.
+  * cli: strip command migrated to docopts.
+  * Obsolete deprecations from 1.x.
+  * Removing the 'vendor' keyword.
+  * Making the icon optional.
+  * cli: snap command migrated to docopts.
+  * Enabling testing with new cli.
+  * Fix ordering for conditional negations in catkin tests.
+  * Replace use of `assert_not_called` in the catkin tests.
+  * General documentation update.
```

Comment:
It comes from a different PR, so mmm, I don't know.
What I think is that from now on, we should make our commit messages better with more context for the changelog. More verbose.